langchain-core/src/output_parsers/string.ts

```diff
@@ -42,4 +47,31 @@ export class StringOutputParser extends BaseTransformOutputParser<string> {
   getFormatInstructions(): string {
     return "";
   }
+
+  _textContentToString(content: MessageContentText): string {
+    return content.text;
+  }
+
+  _imageUrlContentToString(_content: MessageContentImageUrl): string {
```

Comment:
I don't like this actually - would prefer a thrown error instead of silently dropping info